vision/boat/UnitTest.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+import os
+import cv2
+from vision.boat.detectRussianWord import detect_russian_word
+
+class TestDetectRussianWord(unittest.TestCase):      
+    def test_detect_russian(self):
+        """
+        Testing vision.boat.detectRussianWord
+
+        Settings
+        --------
+        detectRussianWord.originalImage: PNG
+            Stores image into a variable.
+        detectRussianWord.text: string
+            Stores text grabbed from image.
+
+        Returns
+        -------
+        True if text grabbed from image matches 'модули иртибот'.
+        False if not.
+        """
+        # Ensure 'модули иртибот' is found within image
+        expected = [1, 1, 1, 1, 1,
+                    1, 1, 1, 1, 1,
+                    0, 0, 0, 0, 0,
+                    0, 0, 0, 0, 0,]
+        matched = []
+        output = []
+        
+        vision_folder = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+        for i in vision_folder:
+            image = cv2.imread(i) 
+            if(detect_russian_word(image)):
+                matched.append(1)
+            else:
+                matched.append(0)
```

Comment:
Lets make expected into a dictionary, {filename: value} so it's robust to changes in the directory.
```
        expected = {
            'boat1.jpg': 1
            'notboat.jpg': 0
        }
        ## You need to run this from IARC-2020/ or vision/
        vision_folder = 'vision' if os.isdir('vision') else ''
        
        for filename, expected_result in expected.items():
            image = cv2.imread(os.path.join(vision_folder, filename))

            result = detect_russian_word(image)

            self.assertEqual(expected_result, result, msg=f"Failed at image {filename}")
```
You can pretty well remove everything else in the function if you add this.